src/book-test-instance.py

```diff
@@ -0,0 +1,393 @@
+#!/usr/bin/env python3
+
+import os
+import json
+import requests
+from requests.auth import HTTPBasicAuth
+from requests_oauthlib import OAuth1
+import sys
+import re
+import argparse
+import tempfile
+import hashlib
+
+
+JIRA_API = 'https://jira.greenpeace.org/rest/api/2'
+GITHUB_API = 'https://api.github.com'
+SWARM_API = 'https://us-central1-planet-4-151612.cloudfunctions.net/p4-test-swarm'
+
+INSTANCE_REPO_PREFIX = 'greenpeace/planet4-test-'
+JIRA_INSTANCE_FIELD = 'customfield_13000'
+
+
+def get_pull_request(pr_number=None, repository=None, pr_url=None):
```

Comment:
Since we'll use this with `pr_url` only, I guess we can simplify it and remove the other args?